skins/laika/src/store/payment/actions.ts

```diff
@@ -52,12 +49,12 @@ export const actions = {
 		context.commit( SET_TYPE, payload );
 		context.commit( SET_TYPE_VALIDITY );
 	},
-	[ setAccountId ]( context: ActionContext<Payment, any>, payload: TypeData ): void {
-		context.commit( SET_BANK_ACCOUNT_ID, payload );
-		context.commit( SET_BANK_ACCOUNT_ID_VALIDITY );
+	[ setBankData ]( context: ActionContext<Payment, any>, payload: BankAccountData ): void {
```

Comment:
Since you need to pass the URL, you can't use the BankAccountData as type here. Depending on where that interface is used, I'd either add the URL field and rename the interface (e.g. `BankAccountRequest` or `BankAccountValidationParams`) or define the type inline:

    [ setBankData ]( context: ActionContext<Payment, any>, payload: { validationUrl: string, bankdata: BankAccountData }): void {